tests/test_cli.py

```diff
@@ -195,7 +196,7 @@ def test_model_3():
         returncode=1,
         stderr=(
             "Error: model=sib is ambiguous, matches (SIBYLL-2.1, SIBYLL-2.3, "
-            "SIBYLL-2.3c, SIBYLL-2.3d)"
+            "SIBYLL-2.3Star-mix, SIBYLL-2.3c, SIBYLL-2.3d)"
```

Comment:
Maybe "SIBYLL-2.3Star-Mix"?

Reply:
This is just the name how it will appear in labels. I prefer the dash here